index.js

```diff
@@ -1,250 +0,0 @@
-// define a range of number
-// set the low and high
-// get mid = low + ((high - low) / 2 )
-// if yes end program
-// if no ask higher or lower
-// change the high , the new high = mid +1  OR!!!
-// OR!!!  change the low , the new low = mid -1
-// get a new mid
-// if yes end program
-// if no repeat line 6 or 7
-const readline = require("readline");
-const readlineInterface = readline.createInterface(
-  process.stdin,
-  process.stdout
-);
-function ask(questionText) {
-  return new Promise((resolve, reject) => {
-    readlineInterface.question(questionText, resolve);
-  });
-}
-
-let numberRange = [];
-let lowNumber;
-let highNumber;
-let midNumber = 0;
-//let welcome = `Hello player! \nGuess a number between ${numberRange[0]} and ${numberRange[highNumber]}`;
-let answerText = "I guess ";
-let startText = "When ready press Y ";
-let answerReply = "If correct press Y otherwise press N\n";
-let startSignal = "Y";
-let tryAgain = "N";
-let celebration = "👍👍👍YAY I got it right!!🎉🎉🎉";
-let secondGuess =
-  "If your guess was higher press H and if it was lower press L\n";
-let numOfSteps;
-let secondSignal = "H";
-let cheatDetector = [];
-let factor;
-let playerAnswer;
-let counter = 0;
-let player;
```

Comment:
Variables all clearly laid out, I also like that you set up repeating strings as variables to call on when needed. Nice use of Emojiis as well 👍